compendium/DeclarativeServices/src/manager/BundleOrPrototypeComponentConfiguration.cpp

```diff
@@ -155,7 +159,10 @@ void BundleOrPrototypeComponentConfigurationImpl::UnbindReference(const std::str
                        std::current_exception());
     }
     
-    context->RemoveFromBoundServicesCache(refName, ref);
+    if (!context->RemoveFromBoundServicesCache(refName, ref)) {
+        GetLogger()->Log(cppmicroservices::logservice::SeverityLevel::LOG_ERROR,
+            "Failure when removing a reference from the BoundServices Cache");
+    }
```

Comment:
Perhaps I'll add a "return" here as well, in case any code is added after the if-statement in the future.